lib/persisted-model.js

```diff
@@ -654,6 +676,22 @@ module.exports = function(registry) {
 
     setRemoting(PersistedModel, 'replaceOrCreate', replaceOrCreateOptions);
 
+    setRemoting(PersistedModel, 'upsertWithWhere', {
+      aliases: ['patchOrCreateWithWhere'],
+      description: 'Update an existing model instance or ' +
+      'insert a new one into the data source based on the ' +
+      'where criteria.',
```

Comment:
+1, please fix the indentation.

@superkhau @Amir-61 I am surprised `eslint` did not catch this problem, would you mind taking a look at our eslint config and see if we are missing a directive?